cpp/include/stream_compaction.hpp

```diff
@@ -71,6 +72,40 @@ gdf_column apply_boolean_mask(gdf_column const &input,
  * @return gdf_column Column containing copy of all non-null elements of @p input.
  */
 gdf_column drop_nulls(gdf_column const &input);
+
+/**
+* @brief Filter only unique row indices in original order
+*
+* @param key_columns cudf::table containing key columns to check duplicates rows
+* @param keep_first keep first or last entry as unique if duplicate is found
+*
+* @return unique_indices in order
+*/
+//rmm::device_vector<gdf_index_type> get_unique_ordered_indices(const cudf::table& key_columns,
+//                           const bool keep_first);
+
+
+/**
+ * @brief Choices for drop_duplicates API for retainment of duplicate rows
+ */
+typedef enum {
```

Comment:
C++ enum. 
```suggestion
enum {
```

Reply:
`typedef enum` is used in src/include/reduction.hpp as well. 
Reason for your suggestion?

Reply:
This is a C-style enum:
```
typedef enum{
} c_enum;
```

This is a C++ enum:
```
enum cpp_enum{
};
```

Use the latter.